ingest/exporter/ingestexportservice.py

```diff
@@ -375,7 +375,7 @@ def bundle_metadata(self, metadata_doc, uuid):
         provenance_core = dict()
         provenance_core['document_id'] = uuid
         provenance_core['submission_date'] = metadata_doc['submissionDate']
-        provenance_core['update_date'] = metadata_doc['updateDate']
+        provenance_core['update_date'] = metadata_doc['dcpVersion']
```

Comment:
Just a note, the metadata team should consider changing `update_date` to `version` in the provenance schema

Reply:
Good point!

Reply:
https://github.com/HumanCellAtlas/metadata-schema/issues/1191